tools/asynccmd/asynccmd.go

```diff
@@ -0,0 +1,176 @@
+package asynccmd
+
+import (
+	"os"
+	"os/exec"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/bitrise-io/go-utils/log"
+)
+
+// Status ...
+type Status struct {
+	Code int
+	Err  error
+}
+
+// Cmd ...
+type Cmd struct {
+	Name string
+	Args []string
+
+	dir     string
+	secrets []string
+	timeout time.Duration
+
+	pid int
+}
+
+// New ...
+func New(name string, args ...string) *Cmd {
+	return &Cmd{Name: name, Args: args}
+}
+
+// SetDir ...
+func (c *Cmd) SetDir(dir string) *Cmd {
```

Comment:
Why `Set*` methods are returning with `*Cmd`? You are not using the return value  when call them in the`EnvmanRun` func.